### ISSUE TITLE
Revisit sixel support?

### ISSUE THREAD

**Issue by cprussin** [thestinger/termite#772]
This ticket re-opens #511 (and expands on #539)

It seems [vte now has sixel support](https://gitlab.gnome.org/GNOME/vte/-/issues/253#note_886802) but [some work must be done to expose it](https://gitlab.xfce.org/apps/xfce4-terminal/-/merge_requests/15/diffs) in terminals that use VTE.

PR inbound to add support behind a config option.

**Comment by cprussin** [comment]
ah, while implementing this I discovered that [the sixel support has been reverted](https://gitlab.gnome.org/GNOME/vte/-/commit/b11351e19e61a5bf07ddad6587af24211dd7e63c).  However, in this revert, the public API functions (most importantly `vte_terminal_set_enable_sixel`) was turned into a stub.  So it should be safe to implement and ship in preparation for vte to ship the filled stub.

So with that I'll still put in a PR to add the option & API call.  Any objections?  @jelly tagging you in for advice as I see you seem to be the most active recent maintainer, sorry if you're not the right person to ask.
